src/nodedef.cpp

```diff
@@ -1311,6 +1311,16 @@ inline void NodeDefManager::fixSelectionBoxIntUnion()
 		m_selection_box_union.MaxEdge.Z / BS - 0.5f);
 }
 
+inline void NodeDefManager::calcBigSelectinBox(content_t id, const ContentFeatures &def)
+{
+	aabb3f box_union;
+	getNodeBoxUnion(def.selection_box, def, &box_union);
```

Comment:
Can't we reuse `m_selection_box_union`?

Reply:
No. `m_selection_box_union` is a union over all already registered nodes.
This should be per node bool.

Reply:
sorry brainfart